workers/unity/ProjectSettings/ProjectSettings.asset

```diff
@@ -47,7 +47,7 @@ PlayerSettings:
   defaultScreenWidthWeb: 960
   defaultScreenHeightWeb: 600
   m_StereoRenderingPath: 0
-  m_ActiveColorSpace: 1
+  m_ActiveColorSpace: 0
```

Comment:
???

Reply:
This was discussed in previous PR - this is intentional change to allow building for both iOS device and simulator